modules/parquet/src/parquetjs/schema/declare.ts

```diff
@@ -13,7 +13,7 @@ export type ParquetCompression =
   | 'LZ4_RAW'
   | 'ZSTD';
 export type RepetitionType = 'REQUIRED' | 'OPTIONAL' | 'REPEATED';
-export type ParquetType = PrimitiveType | OriginalType | DecimalTypes;
+export type ParquetType = PrimitiveType | OriginalType;
```

Comment:
Why are we removing it here... 

'I.e. what is the difference between ParquetType and PrimitiveType, why do we need both?

Should we add JSDoc to these types explaining the difference?

Reply:
`ParquetType` is union of `Primitive` (Physical) and `Original` (Logical) types. The naming and related design came from forked library.

Reply:
It is change of last commit. I revert the change of first commit because it is unnecessary.